draft-tschofenig-rats-psa-token.md

```diff
@@ -365,7 +371,7 @@ The EAT `ueid` (claim key 256) of type RAND is used.  The following constraints
 apply to the `ueid-type`:
 
 * The length MUST be 33 bytes.
-* The first byte MUST be 0x01 (RAND) followed by the 32-bytes key hash.
+* The first byte MUST be 0x01 (RAND) followed by the 32-byte unique identifier of the IAK. {{PSA-API}} provides implementation options for deriving the IAK unique identifier from the IAK itself.
```

Comment:
When you reference the PSA API, are you talking about these two recommendations:

* When using an asymmetric key-pair for the [Initial Attestation Key](https://arm-software.github.io/psa-api/attestation/1.0/about.html#term-initial-attestation-key) (IAK), Arm recommends the Instance ID be a hash of the corresponding public key.

* When using a symmetric key for the IAK, Arm recommends that the Instance ID is always a double hash of the key, hence InstanceID = H(H(IAK)). This eliminates risks when exposing the key to different HMAC block size. For further information, read RFC2104.
    
    
I wonder whether we should reference this draft here for the asymmetric case:
https://datatracker.ietf.org/doc/draft-ietf-cose-key-thumbprint/